bluebutton-data-pipeline-rif-load/src/main/java/gov/hhs/cms/bluebutton/data/pipeline/rif/load/RifLoader.java

```diff
@@ -624,6 +626,9 @@ private void hashBeneficiaryHicn(MetricRegistry metrics, RifRecordEvent<?> rifRe
 				.time();
 
 		Beneficiary beneficiary = (Beneficiary) rifRecordEvent.getRecord();
+		// set the unhashed Hicn
+		beneficiary.setUnhashedHicn(beneficiary.getHicn());
+		// set the hashed Hicn
 		beneficiary.setHicn(computeHicnHash(options, secretKeyFactory, beneficiary.getHicn()));
```

Comment:
Can we also queue up a follow-on PR that renames the old `hicn` column to `hicnHashed` or somesuch? I can see that being REALLY confusing in the future if we don't fix it now.

Reply:
Yes will do...